public/doc/openapi-3-v1.0.0.json

```diff
@@ -456,6 +456,15 @@
                 "description": "Return all approval workflows in ascending sequence order",
```

Comment:
We need to enhance the description to explain the use of resource object for filtering, and possibly with an example.